packages/foundations-ts-definitions/scripts/fetch-definitions.js

```diff
@@ -77,6 +77,13 @@ const fetchSchema = async apiVersion => {
         'api-version': apiVersion,
       },
     },
+    {
+      definitionFile: path.resolve(FOUNDATIONS_TYPES_FOLDER, './organisations-schema.ts'),
```

Comment:
Organisations is a private endpoint so shouldn't be exported as part of @reapit/foundations-ts-definitions

You are right to use the codegen function, can you just remove this line as it will auto-generate a PR each time the API changes and we don't want to publish these types